src/compiler/resolutionCache.ts

```diff
@@ -364,7 +365,7 @@ namespace ts {
                         resolution = resolutionInDirectory;
                     }
                     else {
-                        resolution = loader(name, containingFile, compilerOptions, resolutionHost, redirectedReference);
+                        resolution = loader(name, containingFile, compilerOptions, resolutionHost.getCompilerHost?.() || resolutionHost, redirectedReference);

```

Comment:
Because in case of configured project those updated methods previously were on project which is resolutionCacheHost , we had to get compiler host since those methods are there now.